decidim-core/lib/decidim/asset_router.rb

```diff
@@ -76,5 +76,24 @@ def remote_storage_options
         host: Rails.application.secrets.dig(:storage, :cdn_host)
       }.compact
     end
+
```

Comment:
The entire content of this file should be reviewed after applying #[9597](https://github.com/decidim/decidim/pull/9597) 

The added code is not present there

Reply:
Done.